cmd/verify_test.go

```diff
@@ -143,3 +149,82 @@ func TestHwAttestationPass(t *testing.T) {
 		})
 	}
 }
+
+func TestTdxAttestation(t *testing.T) {
+	file1, err := os.Create("attest")
```

Comment:
Tests should use https://pkg.go.dev/testing#T.TempDirtemp then create files in that temp directory.

Reply:
Should have a more descriptive name like attestFile

Reply:
Thanks for this info. Created a temp dir and renamed it to attestFile